### DISCUSSION TITLE
db.fetch() does not show all results

### DISCUSSION THREAD

**Discussion by deontic** [deta/cloud-docs#436] [General] [answered]
I have 15 items saved with the id "da2e3519-a84d-4b5a-bbb9-9938a3528700" 

<img width="380" alt="image" src="https://user-images.githubusercontent.com/68165727/187730902-ac05e091-9253-412f-9da1-1808cf21b2fc.png">

when I query all objects with that id, there's always only 3 items there.

<img width=390 src=https://user-images.githubusercontent.com/68165727/187731255-554e642f-96ca-42f5-82ae-72262cf67901.png>

 > **3**

Why are there not 15? I have made sure all those objects in the database have the same id, and no matter how many times I re-run the above code it says 3.

**Comment by deontic** [comment]
Pretty sure this is because there might be some limit to how much data it can transfer back to me at one time. How do I get around this? Is it even possible to get more than X KB of results?

**Comment by deontic** [comment]
This might just be solved by providing options to .fetch, I don't know why I didn't think to read the documentation. Will update this after I try it.

**Comment by deontic** [comment]
This might just be solved by providing options to .fetch, I don't know why I didn't think to read the documentation. Will update this after I try it.

**Comment by jnsougata** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/53375272/187734111-fb17fb3f-6019-4003-bb1c-1b4a48f454ab.png)
I ig you just shredded a file into small pieces and stored with same hased-id so when you query it's excedding the 1mb limit

**Reply by deontic** [reply]
>This might just be solved by providing options to .fetch

I was right, that's pretty much it
